src/manage.py

```diff
@@ -3,6 +3,7 @@
 import os
 import sys
 
+#from moiz fixing bug
```

Comment:
I have change the code here

Reply:
Great fix , Thank you for your contributions